integration/examples/test-plugin/gcb/skaffold.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: skaffold/v1beta4
+kind: Config
+build:
+  artifacts:
+  - image: gcr.io/k8s-skaffold/skaffold-example
+    plugin:
+      name: docker-skaffold
```

Comment:
I'm OK with that, but I feel that ideally a user could just write `docker` and it would work.

Maybe skaffold could automatically prepend `skaffold-plugin` to the name, and we could just have `docker` or `bazel` in the `skaffold.yaml`.

It's a bit more confusing when writing plugins, but as long as we were clear in the documentation around how to write a plugin I think it would work. WDYT?